test/specs/components/RadioGroup/RadioGroup-test.tsx

```diff
@@ -147,21 +149,23 @@ describe('RadioGroup', () => {
       expect(checkedValueChanged).not.toHaveBeenCalled()
     })
 
-    it('should not set the value when disabled item is clicked', () => {
-      const wrapper = mountWithProvider(<RadioGroup items={getItems({ disabledItem: 1 })} />)
-      const radioGroupItems = wrapper.find('RadioGroupItem')
+    if (isShorthandApiTest) {
+      it('should not set the value when disabled item is clicked', () => {
```

Comment:
would suggest to replace `value` with `checked` - otherwise description is a bit misleading